client/bootstrap_demo_currency.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-CLIENT="../target/release/encointer-client-notee"
+CLIENT="../target/release/encointer-client-notee -p 9946"
```

Comment:
I don't like any parachain-specific things in here. The script should run with minimal CLI options and I don't want to modify tutorials

Reply:
I agree, which is why this was reverted in 4626f50cf112f47bcbe20d316f4bf8647dbaf876 :)